lib/lightstep/span.rb

```diff
@@ -42,7 +42,7 @@ def initialize(
     )
 
       @tags = Concurrent::Hash.new
-      @tags.update(tags) unless tags.nil?
+      @tags.update(tags.each do |k, v| tags[k] = v.to_s end) unless tags.nil?
```

Comment:
`do .. end` is common in Ruby for multiline blocks, but for inline blocks curly braces should be preferred. So:

```suggestion
      @tags.update(tags.each { |k, v| tags[k] = v.to_s }) unless tags.nil?
```